mlir/lib/Dialect/Rock/IR/RockDialect.cpp

```diff
@@ -859,6 +930,8 @@ LogicalResult GemmOp::verify() {
   }
 
   if (getDerivedBlockSize().has_value() && !isXdlops && !isWmma) {
+    llvm::dbgs() << "FEAUTRES: " << getFeatureValue(this->getOperation())
+                 << "\n";
```

Comment:
Is this print for debugging ? Can it instead be part of error message if you want to keep it